src/booleans.js

```diff
@@ -1,61 +1,62 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return a == 0 && b == 0;
```

Comment:
Should be using strict comparisons throughout this file:
```suggestion
  return a === 0 && b === 0;
```